libs/sdk-ui-charts/package.json

```diff
@@ -65,7 +65,6 @@
         "custom-event": "^1.0.1",
         "date-fns": "^2.8.1",
         "highcharts": "8.2.2",
-        "highcharts-grouped-categories": "^1.1.6",
```

Comment:
Please run `rush update` we should see some changes in `browser-approved-pacakges.json` as this was the only package using the plugin